BUILD.bazel

```diff
@@ -1,4 +1,50 @@
 # TODO(#1532): Rename file to 'BUILD' post-Gradle.
```

Comment:
Will we be replacing all instances of //visibility:public with one of these package groups?

Reply:
No, I don't think so. We may want to replace all instances with _a_ package group, but the ones here are specifically corresponding to the types of libraries introduced in this PR.